drivers/bluetooth/hci/ipm_stm32wb.c

```diff
@@ -420,11 +420,10 @@ static void start_ble_rf(void)
 	LL_RCC_SetRTCClockSource(LL_RCC_RTC_CLKSOURCE_LSI);
 #endif
 
-	/* Set RNG on HSI48 */
-	LL_RCC_HSI48_Enable();
-	while (!LL_RCC_HSI48_IsReady()) {
-	}
-
+	/* HSI48 Clock is enabled through the DTS */
+#if !STM32_HSI48_ENABLED
+#warning Bluetooth IPM requires HSI48 clock to be enabled
+#endif
```

Comment:
Warning is good, but it should not prevent correct behavior. This code section shouldn't be purely removed, only deprecated.
Apply everywhere.

Reply:
Keeping HSI48 clock enable and adding a warning in case this is not defined by the DTS